client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -10,6 +10,8 @@ use test_network::*;
 
 const TRIGGER_NAME: &str = "mint_rose";
 
+// nocheckin why it not work
```

Comment:
While you're at it, see if adding extra diagnostics to this path makes a difference.